py/desispec/pixgroup.py

```diff
@@ -704,6 +719,28 @@ def frames2spectra(frames, pix=None, nside=64):
                 outmap['FIBERSTATUS'] |= banddict[band]['FIBERSTATUS']
         merged_over_cams_fmaps.append(outmap)
 
+    #- Only keep fibermap keywords that apply to combined spectra, not individual exp
+    keep_keywords = ['SURVEY', 'PROGRAM', 'EXTNAME', 'LONGSTRN',
+                     'INSTRUME', 'OBSERVAT', 'OBS-LAT', 'OBS-LONG', 'OBS-ELEV', 'TELESCOP']
+
+    if onetile:
+        keep_keywords.extend([
```

Comment:
I would just maybe move this list outside the if statement into a tile_keywords variable, so this list of keywords is  separated from logic/control flow

Reply:
done, thanks for suggestion